image_view/src/nodes/image_saver.cpp

```diff
@@ -193,16 +193,19 @@ int main(int argc, char** argv)
   image_transport::ImageTransport it(nh);
   std::string topic = nh.resolveName("image");
 
+  ros::NodeHandle local_nh("~");
+  int queue_size;
+  local_nh.param("queue_size", queue_size, 1);
```

Comment:
Can you please make this something like "subscriber_queue_size" so users know exactly which queue size they're setting?

Reply:
@JWhitleyWork 
thanks for the comment, changed `queue_size` variable to `subscriber_queue_size`. I keep param name as `queue_size` since this is `image_saver` and it does not publish any data, and also other node in image_piplne takes `queue_size` rosparm. 
https://github.com/ros-perception/image_pipeline/search?q=queue_size